lm_eval/tasks/yanolja/metrics.py

```diff
@@ -121,7 +182,7 @@ def agg_bartscore_src(items):
         "ja": "ja_XX",
         "zh": "zh_CN"
     }
-    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+    device = torch.device("cuda:1" if torch.cuda.is_available() else "cpu")
```

Comment:
why do we need to specify the index?

Reply:
If we give just "cuda", the huggingface evaluate library utilizes "cuda:0" automatically, which occurs OOM...

Reply:
OOM on a single GPU (cuda:0) can be solved by just specifying the other single GPU? It looks weird ...

Reply:
In this script, BLEURT is first uploaded to cuda:0 (maybe huggingface evaluate default) and BARTScore is also uploaded to cuda:0, so it makes OOM...

Reply:
Such things need to be documented in comments. It looks weird at first glance.